public/fonts/style.css

```diff
@@ -1,3 +1,8 @@
-body {
+html, body {
 	font-family: 'Roboto';
+    font-size: 14px;
 }
+* {
+    line-height: 1.15;
```

Comment:
line height should be variable

Reply:
can't use variable here because it's an static CSS file in the /public folder.